From b7f523e35d9ec927bab67d7a2f10db77bc9fdff5 Mon Sep 17 00:00:00 2001
From: Vernon Yang <vernon2gm@gmail.com>
Date: Tue, 10 Jan 2023 23:22:16 +0800
Subject: [PATCH 63/63] maple_tree: fix comment of mte_destroy_walk

The parameter name of maple tree is mt, so make the comment to correct.

Fixes: 54a611b60590 ("Maple Tree: add new data structure")
Signed-off-by: Vernon Yang <vernon2gm@gmail.com>
---
 lib/maple_tree.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lib/maple_tree.c b/lib/maple_tree.c
index ab99738ad..415f3c413 100644
--- a/lib/maple_tree.c
+++ b/lib/maple_tree.c
@@ -5674,7 +5674,7 @@ static void mt_destroy_walk(struct maple_enode *enode, unsigned char ma_flags,
 /*
  * mte_destroy_walk() - Free a tree or sub-tree.
  * @enode - the encoded maple node (maple_enode) to start
- * @mn - the tree to free - needed for node types.
+ * @mt - the tree to free - needed for node types.
  *
  * Must hold the write lock.
  */
-- 
2.39.0.rc2.1.gbd5df96b79

